src/pages/public/RegistrationForms/MembershipForm/index.js

```diff
@@ -208,7 +208,7 @@ const MembershipFormContainer = (props) => {
       fname: first_name,
       lname: last_name,
       major: memberType === "UBC" || memberType === "UNI" ? major : "",
-      email: email.toLowerCase(),
```

Comment:
@voctory  this is good to go into prod right?

Reply:
Didn't get to test it on dev branch yet

Reply:
Should be fine for prod